csharp/src/Ice/Acm.cs

```diff
@@ -79,15 +79,32 @@ public Acm(TimeSpan timeout, AcmClose close, AcmHeartbeat heartbeat)
             Heartbeat = heartbeat;
         }
 
+        /// <summary>Computes hash code for the current object.</summary>
+        /// <returns>A hash code for the current object.</returns>
         public override int GetHashCode() => HashCode.Combine(Timeout, Close, Heartbeat);
 
+        /// <summary>Determines whether two object instances are equal.</summary>
+        /// <param name="other">The object to compare to.</param>
+        /// <returns><c>true</c> if the specified object is equal to the current object; otherwise, <c>false</c>.
+        /// </returns>
```

Comment:
```suggestion
        /// <summary>Determines whether two object instances are equal.</summary>
        /// <param name="other">The object to compare to.</param>
        /// <returns><c>true</c> if the specified object is equal to the current object, otherwise <c>false</c>.
        /// </returns>
```